src/test/java/org/cactoos/scalar/AvgOfTest.java

```diff
@@ -51,213 +51,313 @@ public void withEmptyCollection() {
     }
 
     @Test
-    public void withIntegerCollection() {
+    public void withIntCollectionIntValue() {
         MatcherAssert.assertThat(
             new AvgOf(
-                new ListOf<>(1, 2, 3, 4).toArray(new Integer[4])
+                1, 2, 3, 4
```

Comment:
@allanws What's is the expected answer when using 1, 3, 7 as input? 1 + 3 + 7 = 11 / 3 = 3,6. But 1, 3 and 7 are integers. The result should be a integer too or a double?